plugin/sdk_v2/public/bus/memory.py

```
@@ -0,0 +1,24 @@
+
+
+class Memory(BusClientBase):
+    def __init__(self, _transport=None):
+        super().__init__(_transport, namespace="memory")
+
+    async def query(self, bucket_id: str, query: str, *, timeout: float = 5.0) -> Result[Any, BusErrorLike]:
+        items = self._state.memory.get(bucket_id, [])
+        matches = [item for item in items if query.lower() in str(item).lower()]
+        return Ok(matches)
+
+    async def get(self, bucket_id: str, *, limit: int = 20, timeout: float = 5.0) -> Result[list[Mapping[str, Any]], BusErrorLike]:
```

Ruff (0.15.5)

[warning] Unused method argument: `timeout`

(ARG002)